src/test/java/de/orchound/gameoflife/model/BoardTest.java

```diff
@@ -18,11 +18,57 @@ void setUp() {
 	}
 
 	@Test
-	void testSetCellAlive() {
+	void testResurrectCellZeroCellIndex() {
```

Comment:
Please don't test each edge case in a dedicated method.
Simply implement `testResurrectCell()` and `testKillCell()`, then put as many assertions in as you might need to test the edge cases.